src/test/java/net/dancier/MainTest.java

```diff
@@ -0,0 +1,32 @@
+package net.dancier;
+
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.core.Response;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class MainTest {
+
+    public static final  DropwizardAppExtension<DancerConfiguration> APP_EXTENSION
+            = new DropwizardAppExtension<>(
+            DancerApplication.class,
+            ResourceHelpers.resourceFilePath("config.yml")
+    );
+
+
+    @Test
+    public void testGetRecommendations() {
+        Response response = APP_EXTENSION.client().target(
+                String.format("http://localhost:%d/recommendations", APP_EXTENSION.getLocalPort()))
+                .request()
+                .get();
+
+        assertThat(response.getStatus()).isEqualTo(200);
```

Comment:
Why not using Junit assert?

Reply:
I just used it here honestly, because I followed an example for writing an Int-Test for dropwizard.
In general I have no strong feelings about what to use, although I slightly thing that assertj statements are a little bit more readable, and also testing for collections is (I think) someway more easy.
If you have more strong feelings, just tell: I will follow you ;-)

Reply:
I would always use AssertJ over JUnit asserts cause they read fluently. And they have a lot of nice helper functions :)

Reply:
I prefer JUnit + Hamcrest (fluent enough for my taste and nice helpers) but not knowing the power of AssertJ. Just very happy with that combination and would be my choice - but would use AssertJ, too.

Reply:
So in the end consent of all for AssertJ. Fine ;-)